core/mount/losetup_linux_test.go

```diff
@@ -29,7 +29,7 @@ var randomData = []byte("randomdata")
 func createTempFile(t *testing.T) string {
 	t.Helper()
 
-	f, err := os.CreateTemp("", "losetup")
+	f, err := os.CreateTemp(t.TempDir(), "losetup")
```

Comment:
```suggestion
	f, err := os.Create(t.TempDir(), "losetup")
```

If we're in a `t.TempDir` we don't need the randomized name from `os.CreateTemp`.

Please also remove `defer`s that delete these files.  `t.TempDir` implicitly adds cleanup functions to `t.Cleanup`.

Reply:
It looks like there are still `defer`s that need to be removed in `TestRoLoop`, `TestRwLoop`, `TestAttachDetachLoopDevice`, `TestAutoclearTrueLoop`, and `TestAutoclearFalseLoop`.

Reply:
Looks like the suggestion to change `os.CreateTemp()` to `os.Create()` has not been handled; agree that we only need one randomized name, not 2 layers of randomization